Starting Block Node fails

### Description

Trying to startup block node grpc server fails with following error:

```
Caused by: java.lang.IllegalAccessException: class com.swirlds.config.impl.internal.ConfigDataFactory (in module com.swirlds.config.impl) cannot access class com.hedera.block.server.consumer.ConsumerConfig (in module com.hedera.block.server) because module com.hedera.block.server does not export com.hedera.block.server.consumer to module com.swirlds.config.impl
```

### Steps to reproduce

1. Run `./gradlew run` command

### Additional context

_No response_

### Hedera network

_No response_

### Version

latest

### Operating system

None

--- a/server/src/main/java/module-info.java
+++ b/server/src/main/java/module-info.java
@@ -2,6 +2,11 @@ import com.hedera.block.server.config.BlockNodeConfigExtension;
 
 /** Runtime module of the server. */
 module com.hedera.block.server {
+    exports com.hedera.block.server.consumer to
+            com.swirlds.config.impl;
+    exports com.hedera.block.server.persistence.storage to
+            com.swirlds.config.impl;
+
     requires com.hedera.block.protos;
     requires com.google.protobuf;
     requires com.lmax.disruptor;

